src/Blog_App_frontend/src/AddBlog.jsx

```diff
@@ -17,7 +18,13 @@ const addBlogSchema = Yup.object().shape({
 });
 
 function AddBlog({ handleAddBlog }) {
-  const navigate = useNavigate(); 
+  const navigate = useNavigate();
+  const queryClient = useQueryClient();
+  const mutation = useMutation(handleAddBlog, {
+    onSuccess: () => {
+      queryClient.invalidateQueries('blogs');
+    },
+  });
```

Comment:
onError show the error and also for onSuccess check if the result is #ok or #err

Reply:
ok